examples/gallery/basemaps/ternary.py

```diff
@@ -0,0 +1,44 @@
+"""
+Ternary diagram
+---------------
+The :meth:`pygmt.Figure.ternary` method can draw ternary diagrams. The example
+shows how to plot circles with a diameter of 0.1 centimeters
+(``style="c0.1c"``) on a 10-centimeter-wide (``width="10c"``) ternary diagram
```

Comment:
For me this example looks good!
One thing I started thinking about, while working on PR 2278, is whether it is 10-centimeter-wide (singular) or 10-centimeters-wide (plural). But this is only a detail.